content/applications/sales/rental.rst

```diff
@@ -0,0 +1,83 @@
+======
+Rental
+======
+
+**Odoo Rental** is a comprehensive solution to manage your rentals.
+
+From a single view, you can send out quotations, confirm orders, schedule rentals, register when
+products are picked up and returned, and invoice your customers.
+
+.. seealso::
+   - `Odoo Rental: product page <https://www.odoo.com/app/rental>`_
+
+Configuring rental pricing
+==========================
+
```

Comment:
By default, when you check your products from the Rental app, there is a filter to only display products that have the "can be rented" box ticked. And when you create a product (still from the app), the product will have that box ticked by default. So that's my reasoning for not including it.
However, I digress, as I still think it could be worthwhile to add that info as a `:tip:` somewhere :)